_alerts/2017-09-28-segwit2x-safety.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Beware of Coinbase, Xapo, and Bitpay!"
+shorturl: "segwit2x-safety"
+active: true
+banner: "Beware of Coinbase, Xapo, and Bitpay!"
+bannerclass: "alert"
+date: 2017-09-28
+---
+
+## Summary
+
+A minority of Bitcoin businesses plan on switching their users to a new incompatible altcoin
+which they plan on wrongly naming Bitcoin through a controversial and contentious hard fork.
+We believe that this new altcoin has no technical legitimacy and will only exist to make it
+easier for corporations to influence the direction of the Bitcoin project. We urge all
+Bitcoin users to immediately remove all their coins from Xapo, Bitpay and Coinbase in particular before November.
```

Comment:
Perhaps instead of listing specific companies here, we should be generic and say "remove all their coisn from such services before November." Then at the bottom of the post we can have a list of companies that are currently known to support segwit2x and something pointing to a statement where they say they will. This way we can remove companies if they change their position.

I think this should be done here and elsewhere "Xapo, Bitpay, and Coinbase" is mentioned in this post.